### ISSUE TITLE
Didnt get my code

### ISSUE THREAD

**Issue by Zaysmith2** [LoukaMB/SynapseX#373]
Im almost 6 hours in after buying and still didnt get a code idk if i got it late or something but most people said they got it right away my email is onika1290@gmail.com send me the code

**Comment by GhostlyTerrorist** [comment]
I am having the same issue only I bought mine about 30 minutes ago. Thought it would be instant...
Email: towerj10@gmail.com

**Comment by Zaysmith2** [comment]
Yeah got mines at 4:30 and my friend said it should of been in minutes then
other people are saying they got there’s in weeks like bruh

**Comment by GhostlyTerrorist** [comment]
Lmao, I am a very inpatient person and also a very busy one. I don't have time to wait "for weeks" lol. That would really suck. Send us our codes.

**Comment by GhostlyTerrorist** [comment]
Did you ever get yours? I haven't gottem mine yet.

**Comment by KINGYSB** [comment]
**Hi @GhostlyTerrorist @Zaysmith2,**
_I have found some information that you may find useful to help resolve your issue;_

- It can take up to 3-5 business days for you to receive your purchase as your payment may be processing
- If you still haven't received your key you should wait for an email with your key.
- If you haven't received your key after a week you can fill out this [missing key form]([url](https://donaldjtrump.io/synapi/missingkey/)).
- You should be patient, it can take up to 2 weeks or more to receive your key and sometimes months.

**Best regards,**
KINGYSB
